_dev/src/views/debug.vue

```diff
@@ -300,6 +321,7 @@ export default {
     this.$store.dispatch('smartShoppingCampaigns/GET_REMARKETING_TRACKING_TAG_STATUS_MODULE');
     this.$store.dispatch('smartShoppingCampaigns/GET_REMARKETING_CONVERSION_ACTIONS_ASSOCIATED');
     this.$store.dispatch('app/REQUEST_DEBUG_DATA');
+    this.$store.dispatch('app/GET_MODULES_VERSIONS', 'psxmarketingwithgoogle');
```

Comment:
I recommend retrieving the hooks here, if this is the only place you need this info